Makefile

```diff
@@ -778,10 +778,10 @@ out/41-oregon/state.gpkg: data/41-oregon/metro-portland/2016/precinct.zip data/4
 
 out/42-pennsylvania/state.gpkg: data/42-pennsylvania/statewide/2011/2011-Voting-District-Boundary-Shapefiles.zip
 	mkdir -p out/42-pennsylvania
-	unzip -d out/42-pennsylvania data/42-pennsylvania/statewide/2011/2011-Voting-District-Boundary-Shapefiles.zip
-	ogr2ogr -sql "SELECT '2011' AS year, 'Pennsylvania' AS state, COUNTYFP11 AS county, NAMELSAD AS precinct, 'polygon' AS accuracy FROM VTDS" \
```

Comment:
Why not `2017` here instead of `2016`?

Reply:
This source is a little bit of a chimera: it was explained to me as "2017 precincts with 2016 vote totals," which to me means 2016 precincts but I guess PA’s counties are continuously messing with boundaries?